api/models/Treatment.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
+
+const validate = require('validate.js');
+
+module.exports = {
+  attributes: {
+    treatmentType: Sequelize.STRING,
+    treatmentFrequency: Sequelize.STRING,
+    treatmentStartDate: Sequelize.STRING,
```

Comment:
Future improvement: Same note about using Date fields. Let's do this before we go to real prod.